android-clang_8/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM conanio/gcc8
```

Comment:
As I understand there is nothing special about `conanio/gcc8` we use it just to get `USER conan`, right?

Reply:
we may use any base image. we need things like CMake, make, autotools, nasm, etc., which are available in the base image.